### PR TITLE
Update rplidar.md with instructions for finding device path on mac OS…

### PR DESCRIPTION
**Pull request by EshaMaharishi**
… M1 and M2

# Description

While testing, I noticed that the device path we document didn't match what was on my system. Jeremy let me know the device path can vary and showed me how to determine the right path for my system.

### CODE REVIEW

File: docs/extend/modular-resources/examples/rplidar.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ Click on the **Modules** subtab. Add the rplidar module with a name of your choi
 {{% tab name="JSON Template" %}}
 
 Navigate to the **Config** tab.
-Select the **Raw JSON** mode, then copy/paste the following `"components"` and `"modules"` JSON:
+Select the **Raw JSON** mode, then copy/paste the following `"components"` and `"modules"` JSON. For M1 & M2, you will need to check your system's output for `ls /dev/ | grep tty.usbserial` and put the number seen there. For example, you may see `tty.usbserial-130`, in which case you should replace XXX with 130:
```
Comment (npentrel) [inline docs/extend/modular-resources/examples/rplidar.md:108]:
```suggestion
Select the **Raw JSON** mode, then copy/paste the following `"components"` and `"modules"` JSON.
If you are on an M1 or M2 Macbook,  run `ls /dev/ | grep tty.usbserial` to determine the device path and use it in your configuration. 
For example, you may see `tty.usbserial-130`, in which case your device path would be `/dev/tty.usbserial-130`:
```

**Comment by npentrel** [comment]
@EshaMaharishi is this ready to merge?

**Comment by EshaMaharishi** [comment]
@npentrel apologies for the delay, it wasn't rendering as I expected on docs-test.viam.dev and I had been meaning to come back to it. When I just tried, I'm not able to render the page at all: https://docs-test.viam.dev/extend/modular-resources/examples/rplidar/

Do you happen to know why that be happening?

**Comment by npentrel** [comment]
It's missing the hash - the links on the staging are not fully functional. I thought I had fixed that but it's still broken... Sorry
https://docs-test.viam.dev/0706798282dfd953804073ef23684fa65dd6df97/public/extend/modular-resources/examples/rplidar/

**Comment by EshaMaharishi** [comment]
@npentrel ahh, gotcha. ok, mind if i look at why it's not rendering in the spot i thought it would and ping you again? sorry about the double review :(

**Comment by EshaMaharishi** [comment]
@npentrel @jeremyrhyde updated so that the message appears on both the Config Builder and JSON Template tabs. Would you mind please taking one more look?

File: docs/extend/modular-resources/examples/rplidar.md [hunk 2]
```diff
@@ -79,7 +82,7 @@ Paste the following into the **Attributes** field of your new component accordin
```
Comment (npentrel) [inline docs/extend/modular-resources/examples/rplidar.md:82]:
```suggestion

If you are on an M1 or M2 Macbook,  run `ls /dev/ | grep tty.usbserial` to determine the device path and use it in your configuration. 
For example, you may see `tty.usbserial-130`, in which case your device path would be `/dev/tty.usbserial-130`:

```

File: docs/extend/modular-resources/examples/rplidar.md [hunk 3]
```diff
@@ -62,7 +62,10 @@ Add a component with type `camera`, model `viam:lidar:rplidar`, and a name of yo
 
 ![adding rplidar component](/extend/modular-resources/add-rplidar/add-rplidar-component-ui.png)
 
-Paste the following into the **Attributes** field of your new component according to your machine's architecture (none needed for Linux):
+Paste the following into the **Attributes** field of your new component according to your machine's architecture (none needed for Linux).
+
+If you are on an M1 or M2 Macbook,  run `ls /dev/ | grep tty.usbserial` to determine the device path and use it in your configuration.
+For example, you may see `tty.usbserial-130`, in which case your device path would be `/dev/tty.usbserial-130`:
```
Comment (npentrel) [inline docs/extend/modular-resources/examples/rplidar.md:68]:
```suggestion
```

File: docs/extend/modular-resources/examples/rplidar.md [hunk 4]
```diff
@@ -105,7 +108,9 @@ Click on the **Modules** subtab. Add the rplidar module with a name of your choi
 {{% tab name="JSON Template" %}}
 
 Navigate to the **Config** tab.
-Select the **Raw JSON** mode, then copy/paste the following `"components"` and `"modules"` JSON:
+Select the **Raw JSON** mode, then copy/paste the following `"components"` and `"modules"` JSON.
+If you are on an M1 or M2 Macbook,  run `ls /dev/ | grep tty.usbserial` to determine the device path and use it in your configuration. 
+For example, you may see `tty.usbserial-130`, in which case your device path would be `/dev/tty.usbserial-130`:
```
Comment (npentrel) [inline docs/extend/modular-resources/examples/rplidar.md:113]:
```suggestion
Select the **Raw JSON** mode, then copy/paste the following `"components"` and `"modules"` JSON:
```

**Pull request merged**